ah-rest-useraccount/src/main/java/br/com/arthub/ah_rest_useraccount/api/v1/exception/EmailInvalidException.java

```diff
@@ -0,0 +1,21 @@
+package br.com.arthub.ah_rest_useraccount.api.v1.exception;
+
+import lombok.Getter;
+
+public class EmailInvalidException extends RuntimeException{
+private static final long serialVersionUID = 1L;
+	
+	@Getter
+	private String message;
+	
+	public EmailInvalidException() {
+		/* Mensagem de erro padrão */
+		super("The username is invalid.");
```

Comment:
Ajustar mensagem padrão de erro. Colocar "the email is invalid" e não "username";